frappe/public/js/frappe/views/formview.js

```diff
@@ -77,15 +77,16 @@ frappe.views.FormFactory = class FormFactory extends frappe.views.Factory {
 	}
 
 	fetch_and_render(doctype, name, doctype_layout) {
+		if (name && name === "new") {
```

Comment:
Let's avoid this? This will break viewing for any doctype with name `new` :woozy_face: Not entirely unlikely.